### DISCUSSION TITLE
File name 'menu' be detected as xml instead of bash

### DISCUSSION THREAD

**Discussion by dlintw** [helix-editor/helix#8970] [Q&A] [answered]
I've a file named 'menu' it is written by bash as a shebang starting like:

```
#!/bin/bash
....

```
In vim, it is detected as 'sh' (:set filetype?)
In hx, it  is detected as 'xml'(:set-language)

**Comment by Agilan989** [comment]
I’m so excited about it because I’m not sure what the plan will be for me
and

Thank you
Best regards
Agilanberan

**Reply by dlintw** [reply]
My workaround is ":set-language bash" every time after open the file.  I don't know how to fix it in the rust code, there are many shebang like variable name in the helix source.

**Comment by dlintw** [comment, marked as answer]
in ~/.config/helix/languages.toml
Add a section to force the filename as bash language.
```
[[language]]
name = "bash"
file-types = [ "menu", ".sh"]
```
